src/Subscriptions/StorageManager.php

```diff
@@ -115,19 +110,13 @@ public function storeSubscriber(Subscriber $subscriber, string $topic): void
         }
     }
 
-    /**
-     * Delete subscriber.
-     *
-     * @return \Nuwave\Lighthouse\Subscriptions\Subscriber|null
-     */
     public function deleteSubscriber(string $channel): ?Subscriber
     {
         $key = self::SUBSCRIBER_KEY.".{$channel}";
-        $hasSubscriber = $this->cache->has($key);
 
         $subscriber = $this->cache->get($key);
 
-        if ($hasSubscriber) {
+        if ($subscriber) {
             $this->cache->forget($key);
         }
```

Comment:
You might want to consider using `return $this->cache->pull(self::SUBSCRIBER_KEY.".{$channel}")` for a one-liner implementaton.